.eslintrc.yml

```diff
@@ -36,8 +40,6 @@ rules:
   # this rule now works but generates a lot of issues with the codebase
   # '@typescript-eslint/member-ordering': error
 
-  '@typescript-eslint/type-annotation-spacing': error
```

Comment:
The update to `eslint-config-prettier` now indicates that this rule is unnecessary (because `prettier` handles this formatting).